build-aux/snap/snapcraft.yaml

```diff
@@ -524,4 +525,6 @@ parts:
     <<:
       - *dynamic-linker
     override-prime: |
-      python3 "${CRAFT_PROJECT_DIR}/build-aux/snap/local/verify-dl.py" "${CRAFT_PRIME}" "/snap/snapd/current/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/${DYNAMIC_LINKER}" ";"
+      "${CRAFT_PROJECT_DIR}/build-aux/snap/local/verify-dl.py" \
+          "${CRAFT_PRIME}" \
```

Comment:
Maybe we could have taken the opportunity to read the environment variable if `--prime` is not passed like done in `patch-dl.py`. Just to make the command line smaller.

Reply:
updated the commit to account for this